src/main/java/org/jabref/logic/exporter/ModsExporter.java

```diff
@@ -43,31 +43,30 @@
 import org.jabref.logic.importer.fileformat.mods.TitleInfoDefinition;
 import org.jabref.logic.importer.fileformat.mods.TypeOfResourceDefinition;
 import org.jabref.logic.importer.fileformat.mods.UrlDefinition;
-import org.jabref.logic.util.FileExtensions;
+import org.jabref.logic.util.FileType;
 import org.jabref.model.database.BibDatabaseContext;
 import org.jabref.model.entry.BibEntry;
 import org.jabref.model.entry.FieldName;
 
 
 /**
- * ExportFormat for exporting in MODS XML format.
+ * TemplateExporter for exporting in MODS XML format.
  */
-class ModsExportFormat extends ExportFormat {
+class ModsExporter extends TemplateExporter {
```

Comment:
I don't know by what you define a TemplateExporter: If it's the one where you can define a html/layout file, then this is not one.

Reply:
I now made `Exporter` an abstract class with the basic fields. Not sure why these special exporter derived from `ExportFormat`.